entities/PotionAleph.java

```diff
@@ -0,0 +1,17 @@
+package entities;
+
+public class PotionAleph extends Consumable {
```

Comment:
Not a requested change, but why "Aleph"?

Reply:
First letter of the Hebrew alphabet (abjad)